### ISSUE TITLE
dribbble API deprecated

### ISSUE THREAD

**Issue by SeungbinBaik** [devxoul/Drrrible#14]
로그만 보셔도 아실 내용이긴 한데 혹시 제가 이상하게 했을지도 몰라서 제가 한 프로세스도 함께 올려요

splash -> 로그인 화면 -> 웹뷰 -> 로그인 시도 실패(제한 횟수 초과) -> 드리블 메인페이지로 이동(웹뷰) -> sign in 선택 -> 구글 로그인 -> 로그인 성공 -> 웹뷰 종료 -> 무한 로딩

`2019-09-27 05:19:30.009 💚 DEBUG UserService.fetchMe():30 - REQUEST: GET /user
2019-09-27 05:19:30.306 💛 WARNING UserService.fetchMe():30 - FAILURE: GET /user (401)
    {
        message = "This version of the API is deprecated. This application must use v2: http://developer.dribbble.com/v2/";
    }
`

**Comment by devxoul** [comment]
Yeah, this app won't work anymore since public APIs are deprecated. Here is the notice about it: https://github.com/devxoul/Drrrible/commit/9da646be36dae6db25f56804ab928784782f07a7
